### PR TITLE
Update 2020-12-21-Stimulus-2-Electronic-Boogaloo.md

### PR DESCRIPTION
**Pull request by mallbritten**
Inserted video images with video link instead of attempted embed.

### CODE REVIEW

File: _posts/2020-12-21-Stimulus-2-Electronic-Boogaloo.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ___
```
Comment (juliusakula) [inline _posts/2020-12-21-Stimulus-2-Electronic-Boogaloo.md:11]: can you change line 6 from `> >` to `>`  for the quote-block thing

**Comment by juliusakula** [comment]
- [ ]  checkbox test 😃

**Comment by juliusakula** [comment]
- [ ] Youtube image thumbnails
- [ ] Redundant ads for ad-blocker (link products in text body)

**Comment by mallbritten** [comment]
- [ ] Recommend revising last few sentences at line 27 (FPV)

File: _posts/2020-12-21-Stimulus-2-Electronic-Boogaloo.md [hunk 2]
```diff
@@ -3,15 +3,15 @@ layout: post
...
+An accessory I need more than anything right now is a [Lavalier Microphone](https://www.amazon.com/dp/B0856V2C5Q/ref=as_sl_pc_tf_til?tag=juliusakula-20&linkCode=w00&linkId=57438679284ee330005ef349588dd5fc&creativeASIN=B0856V2C5Q "Lavalier Microphone on Amazon") for better audio when I take videos. A few months ago I ordered a lav mic - the Movo brand, and it was so terrible I returned it after one use. With two microphones, you would expect Person 1 to be on Channel 1, and Person 2 to be on Channel 2. With the Movo, Person 1 and 2 were on Channel 1, so there was no way to isolate the sound. Imagine doing an interview, but when one person is talking you can hear the other person breathing or drinking water. This Comica has good reviews and I was really impressed with the sound quality and the range. I'll be buying this and have about $300 leftover.
```
Comment (juliusakula) [inline _posts/2020-12-21-Stimulus-2-Electronic-Boogaloo.md:14]: ty

**Comment by juliusakula** [comment]
line 33 -- `"below as well"` they might have ad-block enabled

**Pull request merged**